<launch>
	<arg name="body_id" default="21"/>
	<arg name="hovering_height" default="0.6"/>

	<node name="hover_controller" pkg="controller" type="hover_controller.py" clear_params="true" output="screen">
		<param name="qualysis_id" value="$(arg body_id)"/>
		<param name="height" value="$(arg hovering_height)"/>
	</node>

	<node name="rqt_plot_x" pkg="rqt_plot" type="rqt_plot" args="/controller/plot_data_x"/>

	<node name="rqt_plot_y" pkg="rqt_plot" type="rqt_plot" args="/controller/plot_data_y"/>

	<node name="rqt_plot_z" pkg="rqt_plot" type="rqt_plot" args="/controller/plot_data_z"/>

</launch>
